### PR TITLE
Add release notes for v0.19

### PR DESCRIPTION
**Pull request by cbeighley**
@fnoble @henryhallam @denniszollo @soulcmdc @mfine

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -14,6 +15,28 @@ Contents
...
+-----
+
+#### Core
+
+ * Report 
+ * Fix bug where acquisition could get stuck searching an empty doppler range
+ * Fix buffer overflows when building NMEA messages
+ * Add better checks and thread safety to random number generator
+ * Add GPRMC, GPVTG, and GPGLL NMEA messages
+ * Separate FileIO messaging IDs
```
Comment (denniszollo) [inline RELEASE-NOTES.md]: maybe say "directional"

Comment (mfine) [inline RELEASE-NOTES.md]: Right, directional messages for both FileIO and Settings.

Comment (cbeighley) [inline RELEASE-NOTES.md]: @denniszollo @mfine Fixed

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -14,6 +15,28 @@ Contents
  * [v0.10](#v0.10)
  * [v0.8](#v0.8)
 
+v0.19 <a name="v0.19"></a>
+-----
+
+#### Core
+
+ * Report
```
Comment (fnoble) [inline RELEASE-NOTES.md]: Report what?

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -14,6 +15,28 @@ Contents
  * [v0.10](#v0.10)
  * [v0.8](#v0.8)
 
+v0.19 <a name="v0.19"></a>
+-----
+
+#### Core
+
+ * Report 
+ * Fix bug where acquisition could get stuck searching an empty doppler range
```
Comment (fnoble) [inline RELEASE-NOTES.md]: doppler -> Doppler

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -14,6 +15,28 @@ Contents
...
+ * Fix buffer overflows when building NMEA messages
+ * Add better checks and thread safety to random number generator
+ * Add GPRMC, GPVTG, and GPGLL NMEA messages
+ * Settings and FileIO messages are now directional
+
+#### libswiftnav
+
+ * RAIM position outlier filtering
```
Comment (fnoble) [inline RELEASE-NOTES.md]: Maybe "Receiver Autonomous Integrity Monitoring (RAIM) to filter out bad position solutions"?

**Comment by cbeighley** [comment]
@fnoble Fixed

**Pull request merged**